gossip/minimal/goshawk_test.go

```diff
@@ -163,12 +163,12 @@ func TestValidateSTH(t *testing.T) {
 		t.Errorf("validateSTH(no-current-sth)=%v; want nil", err)
 	}
 	// Back-to-front.
-	o.updateSTH(argonSTH1)
+	o.updateSTH(nil, argonSTH1)
 	if err := o.validateSTH(ctx, &sthInfo2); err != nil {
 		t.Errorf("validateSTH(reversed)=%v; want nil", err)
 	}
 	// Valid proof between STHs.
-	o.updateSTH(argonSTH2)
+	o.updateSTH(argonSTH1, argonSTH2)
```

Comment:
Is there an invalid proof test case? hard to tell with github.

Reply:
I think so at l.187